dashboard/config/foorm/library/surveys/pd/ayw_module_1.0.json

```diff
@@ -10,64 +10,7 @@
-      },
-      {
-       "type": "matrix",
-       "name": "module_1",
-       "title": "{workshop_course}: Module 1 Activities",
-       "columns": [
-        {
-         "value": "1",
-         "text": "N/A"
-        },
-        {
-         "value": "2",
-         "text": "1"
-        },
-        {
-         "value": "3",
-         "text": "2"
-        },
-        {
-         "value": "4",
-         "text": "3"
-        },
-        {
-         "value": "5",
-         "text": "4"
-        },
-        {
-         "value": "6",
-         "text": "5"
-        },
-        {
-         "value": "7",
-         "text": "6"
-        },
-        {
-         "value": "8",
-         "text": "7"
-        }
-       ],
-       "rows": [
-        {
-         "value": "pre_work_reading",
-         "text": "(pre-work) Reading and comparing the \"standard\" lesson plan to the \"model\" lesson plan"
-        },
-        {
-         "value": "virtual_meeting_checkin",
-         "text": "(virtual meeting) Checking in on how things are going in the classroom. "
-        },
-        {
-         "value": "virtual_discuss_teaching_practices",
-         "text": "(virtual meeting) Discussing the teaching practices from the model lesson"
-        },
-        {
-         "value": "virtual_discuss_engagement",
-         "text": "(virtual meeting) Discussing how to counsel students who are disengaging or becoming more passive in class."
-        }
-       ]
+       "html": "<h2>How useful was Module 1?</h2>\n\n<p><span style=\"font-family: arial, helvetica, sans-serif;\">For each activity in this module, we ask that you <strong>rate how useful that activity was to you <em>personally</em></strong> in terms of your own <strong>learning, development, teaching practice</strong>, etc. </span></p>"
```

Comment:
Do you mean did I intentionally remove that set of questions? I think the answer is yes, let me confirm with Laura though. She also removed the same set of questions for module 2, I believe.

Reply:
ah ok I was slightly confused because it looked like you kept the prompt but removed the question, but it looks like prompt actually goes with the next question as well

Reply:
Yep, I think that's right.